src/components/card/CardSidebar.vue

```diff
@@ -330,6 +390,14 @@ export default {
 		hideEditor() {
 			this.descriptionEditing = false
 		},
+		showAttachmentModal() {
+			this.modalShow = true
+		},
+		addAttachment(attachment) {
+			const string = '![📎 ' + attachment.data + '](' + this.attachmentUrl(attachment) + ')'
```

Comment:
Append this to the existing description with a newline before inserting the attachment. IIrc we also had a check for the mimetype in the old version and just added a link (no embedded image) for other filetypes.